h2o-core/src/main/java/hex/Model.java

```diff
@@ -2880,6 +2880,8 @@ public boolean testJavaScoring(Frame data, Frame model_predictions, double rel_e
           // Make a prediction
           AbstractPrediction p;
           try {
+            if (rowData.size() < 34)
```

Comment:
should be present in production code?

Reply:
absolutely not!  I am embarrassed...  Thanks for catching it.